recipe/meta.yaml

```diff
@@ -45,12 +46,12 @@ requirements:
     - proj4
     - udunits2
     - zlib
-    - flex
   run:
     - blas 1.1 {{ variant }}  # [not win]
     - bzip2
     - cairo  # [linux]
     - curl
+    - flex
```

Comment:
I thought that flex was needed only at build time.

Reply:
I thought so too, but we've gotten reports of a missing `libfl.so` from users trying to run the existing build, and installing `flex` fixes it.

However, I just pushed two new commits that revert the addition of `flex` as a runtime dependency and instead explicitly links against `libfl.a`. If it works, this should be a cleaner solution.